src/backend/app/Console/Commands/MpmModelsCommand.php

```diff
@@ -74,8 +74,8 @@ public function handle() {
             }
         }
 
-        $this->dateClass = class_exists(\Illuminate\Support\Facades\Date::class)
-            ? '\\'.get_class(\Illuminate\Support\Facades\Date::now())
+        $this->dateClass = class_exists(\Illuminate\Support\Facades\Carbon::class)
```

Comment:
I don't think this is correct. From what I see a `Carbon` Facade doesn't exist. Only a date one. Maybe a copy/paste here?